Source/MyProject2/FlyingPawn.cpp

```diff
@@ -89,17 +92,29 @@ void AFlyingPawn::SetupPlayerInputComponent(class UInputComponent* PlayerInputCo
 }
 
 void AFlyingPawn::VerticalInput(float Input) {
-	// Calculate change in rotation this frame
-	FRotator DeltaRotation(-1 * Input * VerticalTurnSpeed * GetWorld()->GetDeltaSeconds(), 0, 0);
-
-	// Rotate plane
-	AddActorLocalRotation(DeltaRotation);
-
+	FRotator CurrentRotation = GetActorRotation();
+	if (Input != 0) {
```

Comment:
Just mentioning this but they recommend using a class called bIsAlmostZero or something like that, so that joysticks have a little bit of give on their center axis